opentelemetry-api/src/opentelemetry/trace/status.py

```diff
@@ -22,12 +22,12 @@
 class StatusCode(enum.Enum):
     """Represents the canonical set of status codes of a finished Span."""
 
-    OK = 0
-    """The operation has been validated by an Application developer or Operator to have completed successfully."""
-
-    UNSET = 1
+    UNSET = 0
     """The default status."""
 
+    OK = 1
+    """The operation has been validated by an Application developer or Operator to have completed successfully."""
+
```

Comment:
Enum order as followed in [OTLP v0.6.0](https://github.com/open-telemetry/opentelemetry-proto/pull/224/files) and [Spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-status).

Reply:
@codeboten 
We should probably change our status code mappings to also follow this enum right?

Reply:
I would have thought this would've broke some tests in contrib that compared the enum numerical value, but I guess not!